Relevant error messages should be displayed even when --debug flag is not specified

**Describe the bug**
When an error is returned from the Terraform bridge, that error is not shown unless --debug is specified. Instead, an unhelpful error message is shown like visible here:
```
~/git/lyra/build/bin/lyra apply test
2019-05-21T17:56:38.959+0200 [ERROR] lyra: apply failed: Error="Go function Create returned error '{'s' => undef}' (file: workflows/test.yaml)"

▸ apply failed test
```

**To Reproduce**
Steps to reproduce the behavior:
1. Write a workflow like 
```
steps:
  aws_db_group:
    Aws::Db_subnet_group:
      name: test
      subnet_ids:
        - subnet-42ec6835
```
2. Run command `lyra apply test`
3. See error

**Expected behavior**
A clear and concise description of the error should be displayed.

**Logs**
https://gist.github.com/timidri/3868ee1265e0005186fe3efd0bb76bb8

In the log you can see the real error message which should be displayed:
```
2019-05-21T17:54:35.172+0200 [DEBUG] lyra.aws: Invoke failed: @module=aws error="Go function Create returned error 'Error creating DB Subnet Group: DBSubnetGroupDoesNotCoverEnoughAZs: DB Subnet Group doesn't meet availability zone coverage requirement. Please add subnets to cover at least 2 availability zones. Current coverage: 1
	status code: 400, request id: 730a7e43-33ca-4f66-b694-d73fba7577d7' (file: /Users/dimitri.tischenko/go/pkg/mod/github.com/lyraproj/pcore@v0.0.0-20190516164225-2c1838ece043/types/function.go, line: 104)
```
**Environment (please complete the following information):**
 - OS: [e.g. macOS 10.14.3]
 - Version/Build number [1b9588ee878ced42d8356765a5ed60240f171f88]
 - Cloud target [AWS]

--- a/go.mod
+++ b/go.mod
@@ -6,7 +6,7 @@ require (
 	github.com/hashicorp/go-plugin v0.0.0-20190220160451-3f118e8ee104
 	github.com/lyraproj/data-protobuf v0.0.0-20190329160005-a909d9e1f93b
 	github.com/lyraproj/issue v0.0.0-20190513084509-faf9b542f594
-	github.com/lyraproj/pcore v0.0.0-20190516164225-2c1838ece043
+	github.com/lyraproj/pcore v0.0.0-20190521101211-2b39bcf82658
 	github.com/lyraproj/semver v0.0.0-20181213164306-02ecea2cd6a2
 	github.com/stretchr/testify v1.3.0
 	golang.org/x/net v0.0.0-20190311183353-d8887717615a

--- a/service/error.go
+++ b/service/error.go
@@ -91,7 +91,14 @@ func errorFromReported(c px.Context, err issue.Reported) serviceapi.ErrorObject 
 	if len(keys) > 0 {
 		args := make([]*types.HashEntry, len(keys))
 		for i, k := range keys {
-			args[i] = types.WrapHashEntry2(k, px.Wrap(c, err.Argument(k)))
+			av := err.Argument(k)
+			var arg px.Value
+			if ea, ok := av.(error); ok {
+				arg = types.WrapString(ea.Error())
+			} else {
+				arg = px.Wrap(c, av)
+			}
+			args[i] = types.WrapHashEntry2(k, arg)
 		}
 		ds = append(ds, types.WrapHashEntry2(`arguments`, types.WrapHash(args)))
 	}

